src/com/comeet/data/DataRepository.java

```diff
@@ -97,10 +103,65 @@ public Room retrieveRoomMetadata(String email) {
 
         } catch (Exception e) {
             // e.printStackTrace();
-            ApiLogger.logger.log(Level.SEVERE, "Error getting rooms from database database", e);
+            //ApiLogger.logger.log(Level.SEVERE, "Error getting rooms from database database", e);
```

Comment:
I am trying to have all logging occur in the UserService class. That way we don't log the same exception multiple times as it bubbles up to the top